tests/components/notify/test_command_line.py

```diff
@@ -53,6 +53,7 @@ def test_command_line_output(self):
 
             self.hass.services.call('notify', 'test', {'message': message},
                                     blocking=True)
+            self.hass.block_till_done()
```

Comment:
I didn't want to add `block_till_done()` here because passing in `blocking=True` should ensure that the service call has been processed before this method returns